client/components/Login/index.jsx

```diff
@@ -81,7 +85,7 @@ class Login extends Component {
         }
 
         return (
-            <Form className="login__form">
+            <Form className="login__form" onSubmit={this.onLogIn}>
```

Comment:
Future heads up for the team - don't use "click" on submit buttons with forms, use "onSubmit" on the form and a submit button, this ensures other ways of "submitting" (pressing enter in a standard text field) still submit the form